game/CMakeLists.txt

```diff
@@ -106,6 +119,11 @@ set(SELF_BUILT_JSON "NEVER" CACHE STRING "rule to allow self build of Json depen
 set(Aubio_REQUIRED_VERSION "0.4.9")
 
 # Find all the libs that don't require extra parameters
+
+if (MINGW)
+target_link_libraries(performous PRIVATE mingw32)
```

Comment:
Forgot an indent and shouldn't we link against mingw64?

Reply:
No. The library/headers are called mingw32 mingw-w64 is the name of the project that forked the original mingw32 code. This still comes from the `x86_64` version of the package.

Reply:
Alrighty thanks for clarifying!, Then just fix the indent :)